packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -480,6 +517,59 @@ describe('SEPConfiguratorResource', () => {
         .rejects
         .toThrowError(/Failed to save Spot Event Plugin Configurations/);
     });
+
+    test('throw when cannot save group collection', async () => {
```

Comment:
I think these tests can be improved by using something like `sinon` to stub out or mock methods for us in a nicer way. For example, we could rewrite the mocking portion of this test to something like:
```ts
    test('throw when cannot save group collection', async () => {
      // GIVEN
      const stubGetGroupPoolCollection = sinon.stub(SpotEventPluginClient.prototype, 'getGroupPoolCollection').resolves([]);
      const stubSaveGroupPoolCollection = sinon.stub(SpotEventPluginClient.prototype, 'saveGroupPoolCollection').resolves(false);
      const stubConfigureSpotEventPlugin = sinon.stub(SpotEventPluginClient.prototype, 'configureSpotEventPlugin').resolves(false);

      // WHEN
      // ...the rest of the test...
    });
```

With the way it's written right now, we won't be able to do this without changing how all the tests work, so I'd say we can leave this as is. Just wanted to make note of this so that when we need to add more to these tests, we can do a bit of cleanup then.